website/docs/config_guide.md

```diff
@@ -148,6 +148,27 @@ jbrowse add-assembly myfile.fa --load copy --out /var/www/html/jbrowse2
 
 See our CLI docs for the add-assembly for more details here -- [add-assembly](cli#jbrowse-add-assembly-sequence)
 
+Note: assemblies can also be added graphically using the assembly manager. See the quickstart guide for more details. (TODO: link to quickstart)
```

Comment:
to link you can use the syntax

[text of the link](quickstart_web)

where quickstart_web is the id at the top of the quickstart_web.md. You can also link to subsections if needed in the files by adding quickstart_web#subsection 

you can find the subsection name by looking at what happens when you click a section heading in the browser

Reply:
Yup, forgot to add this one